File: Leet Code/Python/problem_50.py
class Solution(object):
    def myPow(self, x, n):
        """
        :type x: float
        :type n: int
        :rtype: float
        """
        result = (float)(pow(x, n))
        return result

mySol = Solution()
output = mySol.myPow(2.00000, 10)
print(output)

File: Leet Code/Java/Problem50.java
public class Problem50 {
    public static void main(String[] args) {
        double numPow = myPow(2.0000, 10);

        System.out.println(numPow);
        
    }

    static double myPow(double x, int n) {
        double result = Math.pow(x, n);
        return result;
    }
}
